prow/plugins/trigger/pr.go

```diff
@@ -86,7 +86,7 @@ I understand the commands that are listed [here](https://github.com/kubernetes/t
 %s
 </details>
 `
-	comment := fmt.Sprintf(commentTemplate, pr.User.Login, trustedOrg, trustedOrg, plugins.AboutThisBot)
+	comment := fmt.Sprintf(commentTemplate, pr.User.Login, trustedOrg, trustedOrg, plugins.AboutThisBotWithoutCommands)
```

Comment:
I think "I understand the commands that are listed here." should appear in the details section so we should keep this line as is and instead remove the phase from `commentTemplate`. WDYT?

Reply:
I also thought that would make more sense. But then I saw that we specifically moved it above the fold in https://github.com/kubernetes/test-infra/pull/2844. So, I didn't change it.

Reply:
Hmmm @crimsonfaith91 can we move the link to the bot commands for the needs-ok-to-test messages back into the details section for consistency with other comments? Besides consistency, linking to the available bot commands is not actually related to the content of the message so it seems like something that should be in the details section.

Reply:
@grodrigues3 Will it be fine to move the link to bot command documentation of needs-ok-to-test message back into `Details` section? @cjwagner has a good argument doing so.

Reply:
No reason to have repetition.  Makes sense to me to get rid of it in one of the places.  I'd prefer to keep it outside the details section (that is, keep it in the main comment), but I don't have a strong opinion one way or another.

Reply:
@cjwagner I also do not have any strong opinion. Feel free to decide which way works best for the users. Thanks for working on this!

Reply:
Lets respect the decision made in #2844 for now then and keep it above the fold.
/lgtm